src/renderer/components/_atoms/react-flow/custom-nodes/contact.tsx

```
@@ -239,33 +248,38 @@
         style={{ width: DEFAULT_CONTACT_BLOCK_WIDTH, height: DEFAULT_CONTACT_BLOCK_HEIGHT }}
       >
         {contact.svg(wrongVariable)}
-        <div className='absolute -left-[24px] -top-[26px] flex h-3 w-[72px] items-center justify-center'>
-          {isEditing ? (
+        <div
+          className='-z-1 pointer-events-none absolute -left-[24px] -top-[24px] w-[72px] overflow-y-scroll [&::-webkit-scrollbar]:hidden'
+          ref={highlightDivRef}
+        >
+          <div
+            className='h-full w-full whitespace-pre-wrap break-words text-center text-xs leading-3 text-transparent'
+            dangerouslySetInnerHTML={{ __html: formattedContactVariableValue }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)